superset/dashboards/api.py

```diff
@@ -152,9 +156,10 @@ class DashboardRestApi(BaseSupersetModelRestApi):
         "owners": ("first_name", "asc"),
     }
     related_field_filters = {
-        "owners": RelatedFieldFilter("first_name", FilterRelatedOwners)
+        "owners": RelatedFieldFilter("first_name", FilterRelatedOwners),
+        "created_by": RelatedFieldFilter("first_name", FilterRelatedOwners),
     }
-    allowed_rel_fields = {"owners"}
+    allowed_rel_fields = {"owners", "created_by"}
```

Comment:
The new accepted `column_name` on the related endpoint will actually fetch the same data has `owners` but it's probably more coherent to add it anyway, and we may want to filter just one of them in the future (not very probable). Anyway, looks good